database/models/section.ts

```diff
@@ -7,10 +7,23 @@ enum SectionType {
     SHORTANSWER = "sa",
 }
 
+enum PaddingType {
+    px = "px",
+    percent = "percent",
```

Comment:
Rather than using "percent" use "%" that way we can use the variable directly when referencing on the front end